static/sidebar-single/js/grid.js

```diff
@@ -145,6 +145,40 @@
           }, '*');
         }
       };
+      var ajustElements = function(className,property,value){
+          var elements =document.getElementsByClassName(className);
+            for (var i = 0; i < elements.length; i++) {
+              switch(property){
+                  case "width":
+                    elements[i].style.width = value;
+                  break;
+                  case "display":
+                    elements[i].style.display = value;
+                  break;
+                  case "padding":
+                    elements[i].style.padding = value;
+                  break;
+                  default:
+                  //nothing
+                  break;  
+              }
+            }
+      };
+      var elementWidth = document.getElementsByClassName("tvp-sidebar-container");
+      var currentWidth = elementWidth.length>0 ? elementWidth[0].offsetWidth : 0;
+      if(currentWidth>0 && currentWidth <= 205){
+            ajustElements("tvp-video-image","width", "100%");
+            ajustElements("tvp-video-details","width", "100%");
+            ajustElements("tvp-video-views","display", "none");
+            ajustElements("tvp-video-author","display", "none");
+            ajustElements("tvp-video-details","padding", "5px 0 0 0");
+      }else{
+            ajustElements("tvp-video-image","width", null);
```

Comment:
I think you can save some code here, looks like the "else" code block is useless.

if `null` is passed to the `switch` statement it will go to the defult case which does nothing.
We can avoid that if you remove the "else" code block